tests/test_speed.py

```diff
@@ -136,7 +146,7 @@ async def make_remote_and_topic_writer(
                 await asyncio.wait_for(process.wait(), timeout=STD_TIMEOUT)
         finally:
             if process.returncode is None:
-                print("Warning: killing the topic writer")
+                self.log.debug("Warning: killing the topic writer")
```

Comment:
I would *much* rather that you leave the print statements alone. When I run this test with `pytest -v tests/test_speed.py -s` to see the data locally I don't want to have to specify a hard-to-remember flag such as `--log-cli-level 20` (both the name and the level are non-obvious, whereas I have `-s` hard-wired).

The *point* of this test is to *print results* (and, if possible, publish them). So print statements are arguably correct here.

I tried mixing log messages and print statements and the results seemed fine to me. They are output in the right order, which is the most important thing to my mind. I realize you are not as happy with it, but I'm not sure why.

Note: I could also live with log messages that are reliably output. But you objected to making `--log-cli-level 20` the default (in setup.cfg). That's fine with me. I prefer print statements in unit tests anyway, because they simpler, less work, and I *know* they will be output.